javascript-light-bulb/main.js

```diff
@@ -0,0 +1,17 @@
+var clickCount = 1;
+
+var $circle = document.querySelector('.circle');
+var $body = document.querySelector('body');
+
+function handleClickEvent(event) {
+  clickCount++;
```

Comment:
Rather than using a counter variable then checking if the value is even or odd, I would suggest using a boolean to keep track of if the light is currently on or off, and re-assign the value to the opposite boolean every time the user clicks the light switch.